jupyter-extension/src/plugins/mount/components/Config/__tests__/Config.test.tsx

```diff
@@ -133,12 +135,23 @@ describe('config screen', () => {
         />,
       );
 
-      await getByTestId('Config__pachdAddressUpdate').click();
-      expect(getByTestId('Config__mountConfigSubheading')).toHaveTextContent(
-        'Update Configuration',
-      );
-      await getByTestId('Config__pachdAddressCancel').click();
-      expect(getByTestId('Config__pachdAddress')).toHaveTextContent(
+      //console.log(await findByTestId('Config__pachdAddressUpdate'));
```

Comment:
I'm guessing you probably want to delete this

Reply:
good catch, i do indeed